src/final2/subtests/InvalidCommandTest.java

```diff
@@ -155,17 +155,11 @@ public void createExistingAntTest() {
 	@Test
 	public void invalidAntNameTest() {
 		for (String invalidAntName : INVALID_ANT_NAMES) {
-			inputFile = new String[] {
-					"0000",
-					"0000",
-					"0a00",
-					"0000"
-			};
 			runs = new Run[] {
 					new ErrorRun("create " + invalidAntName + ",1,1"),
 					quit()
 			};
-			sessionTest(runs, Input.getFile(inputFile));
+			sessionTest(runs, Input.getFile(TASK_SHEET_INPUT_FILE_1));

```

Comment:
> In TASK_SHEET_INPUT_FILE_1, cell 1,1 is "empty" and not an obstacle

But I don't see this (immidiatly) in the test. I think it's best, when you can understand every test without knowing anything about TASK_SHEET_INPUT_FILE_1 (except that it's an valid playing field). (Due to clarity)